tests/test_lab_config.py

```diff
@@ -16,3 +20,93 @@ def test_ilab_config_show(cli_runner: CliRunner) -> None:
     assert parsed
 
     assert configuration.Config(**parsed)
+
+
+def test_ilab_config_init_with_env_var_config(
+    cli_runner: CliRunner, tmp_path: pathlib.Path
+) -> None:
+    # create a dummy config file
+    cfg = configuration.get_default_config()
+    assert cfg.general.log_level == "INFO"
+    cfg.general.log_level = "DEBUG"
```

Comment:
(No action required) This test assumes that the default is NOT "debug". Could we pick some other setting in the config that surely won't ever have our custom value as the default? I dunno, set `chat.max_tokens` to `123456`. (This will also allow us to NOT import `logging` to access the `DEBUG` symbol.)

Reply:
Pff. Nevermind, I just noticed the `assert INFO` above. Then at least the test case won't silently become no-op if we ever change the default. But the other argument stands I think.